tests/bson/bson-fromPHP-001.phpt

```diff
@@ -44,20 +45,28 @@ echo "Testing top-level objects\n";
 foreach ($tests as $test) {
     try {
         echo toJson(fromPHP(new MyDocument($test))), "\n";
+    } catch (Exception $e) {
+        printf("%s: %s\n", get_class($e), $e->getMessage());
+    }
```

Comment:
Since separate exceptions are thrown here, it was necessary to wrap each `fromPHP()` call in its own try/catch.

While working on this PR, I had briefly used InvalidArgumentException for the PackedArray error, which is why I ended up changing the catch to a generic Exception and printing the class. That's no longer necessary but I think we can keep it as-is since it will lead to more helpful test failure diffs should the exception types ever change.